recipes/bwa2/2.0pre2_npg/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+n=`expr $CPU_COUNT / 4 \| 1`
```

Comment:
`n="$CPU_COUNT"` is more idiomatic because we mostly build on dedicated hosts where we own all CPUs